public/search.js

```diff
@@ -46,9 +46,39 @@ $('#search').click(function()
     $('#next').css({'visibility': 'visible'});
     $('#prev').css({'visibility': 'visible'});
     displayDocs(firstDoc, lastDoc);
+    
+    var dropDownList = document.getElementById('dropdown');
+    var selectedColl = dropDownList.options[dropDownList.selectedIndex].value;
+    $.ajax(
+    {
+      //url: 'http://localhost:3000/v1/resources/temporal-range?rs:collection='+selectedColl,
+      url: 'http://localhost:3000/v1/resources/temporal-range?rs:collection=myTemporal' ,
+      success: function(response, textStatus) 
+      {
+        displayAxis(response);
+      },
+      error: function(jqXHR, textStatus, errorThrown) 
+      {
+        console.log('problem');
+      }
+    });
   }
 );
 
+//function 
+function displayAxis(times)
+{
+  sysStart = times.sysStart;
+  sysEnd = times.sysEnd;
+  valStart = times.valStart;
+  valEnd = times.valEnd;
+  console.log(sysStart);
```

Comment:
If these console.log statements aren't needed any more, please remove them. If they are, please add something to the output to give some context. console.log('displayAxis() sysStart: ' + sysStart);